index.html

```diff
@@ -1226,6 +1244,52 @@
       <a href="#steps-receiving">Receiving and parsing content</a>
       and <a>Writing or pushing content</a> sections.
     </p>
+    <p>
+      To <dfn>convert an NDEFRecord object's bytes</dfn> to a given <var>type</var>, run these steps:
+    </p>
+    <ol>
+      <li>Let <var>bytes</var> be the <a>NDEFRecord</a> object's associated <a>bytes</a>.
+      </li>
+      <li>Let <var>recordType</var> be the value of <a>NDEFRecord</a> object's recordType attribute.
+      </li>
+      <li>Switch on <var>type</var>:
+        <dl>
+          <dt>text</dt>
+          <ol>
+            <li>
+              If the <var>recordType</var> value is not equal to <code>"empty"</code>, then
+              return the result of running <a>UTF-8 decode</a> on the <var>bytes</var>.
```

Comment:
on bytes should be fine... removing "the"

Reply:
Thanks, fixed.